pages/docs/Style Guide.md

```diff
@@ -0,0 +1,108 @@
+- ♿ [Accessibility](accessibility)
+
+## Inclusive Language 🌎
+
+Inclusive language is an essential aspect of communication. It promotes equality, respects diversity, and avoids offense. Here are some best practices to follow:
+
+- Use gender-neutral words for common terms
+  - ❌ **Not this:** A maintainer should try to be kind yet constructive in **his** feedback.
+  - ✅ **Use this:** A maintainer should try to be kind yet constructive in **their** feedback.
+
+- Avoid using terms that have colonialistic or racist connotations.
+  - ❌ **Not this:** Contributors should push their pull requests to the **master** branch.
+  - ✅ **Use this:** Contributors should push their pull requests to the **main** branch.
+
+## Grammar & Mechanics 📝
+
+Grammar and mechanics help ensure that your writing is clear, concise, and easy to understand. Here are some best practices to follow:
+
+### Commas
+
+- Use commas when describing lists.
+  - ❌ **Not this:** You can also add your **timeline testimonials and upcoming events** that you are participating in.
+  - ✅ **Use this:** You can also add your **timeline, testimonials, and upcoming events** that you are participating in.
+
+- Use commas in introductory elements.
+  - ❌ **Not this:** **If you do not have one yet** you can create one for free with an email address and password.
+  - ✅ **Use this:** If you do not have one yet, you can create one for free with an email address and password.
+
+- Use commas before the seven, so-called, coordinating conjunctions (and, but, so, for, nor, yet, or) to separate two independent clauses.
+  - ❌ **Not this:** There are 4 ways you can add your profile **but** for this Quickstart, we will use the GitHub UI.
+  - ✅ **Use this:** There are 4 ways you can add your profile, **but** for this Quickstart, we will use the GitHub UI.
+
+### Capitalisation
+
+- Only capitalize terms that describe a product, its feature, the first letter of a sentence, or a person’s name.
+  - ❌ **Not this:** There are 4 ways you can add your profile but****for this **quickstart**we will use the **github ui**.
+  - ✅ **Use this:** There are 4 ways you can add your profile, **but** for this Quickstart, we will use the **GitHub UI**.
+
+## Clarity 🤔
+
+Clarity in writing helps ensure that your audience understands your message and intentions. Here are some best practices to follow:
+
+### Active vs. Passive
+
+Using the active voice in writing makes your message more direct and engaging.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+- ✅ **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+### Jargon
+
+Using technical jargon can alienate non-technical readers.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
```

Comment:
seems like this example here is not the example of Jargon

Reply:
Good point @Neha! Do you have an other suggestions?